tests/test_excluded_packages.py

```diff
@@ -0,0 +1,34 @@
+import os
+import subprocess
+
+from common.command_common import build_command
+from common.report_common import assert_story_points_from_report_file
+
+
+def test_excluded_packages(analysis_data):
+    application_data = analysis_data['jee_example_app']
+
+    report_path = os.getenv('REPORT_OUTPUT_PATH')
```

Comment:
If this is always constant , this part can also be done in build_command method .

Reply:
It is already in the build_command method, but it's needed also here to assert the generated files later.